src/utils/config.ts

```diff
@@ -1,12 +1,13 @@
 import { dequal } from 'dequal/lite'
 
 import { wrapCache } from './cache'
-import webPreset from './libs/web-preset'
+import webPreset from './web-preset'
 import { slowConnection } from './env'
-import { Configuration, RevalidatorOptions, Revalidator } from './types'
+import { Configuration, RevalidatorOptions, Revalidator } from '../types'
+import { UNDEFINED } from './helper'
 
 const fetcher = (url: string) => fetch(url).then(res => res.json())
-const noop = () => {}
+const noop = () => false
```

Comment:
not sure if we should change noop to return `false`, if there's a callback return defaultly to true but used noop by accident.

or maybe `const noop = (defaultValue) => () => defaultValue`

Reply:
It's a hack to save some bytes... gonna revert it back. 👍